### PR TITLE
fix(melos): fix melos format script

### PR DESCRIPTION
**Pull request by mhadaily**
## Description

It seems format command is failing hope this is fixing this script. 

## Related Issues

https://github.com/fluttercommunity/plus_plugins/issues/223

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Comment by mhadaily** [comment]
Mm, it's interesting it doesn't throw an error locally but it throws an error on CI. I need to investigate more.

File: .github/workflows/all_plugins.yaml [hunk 1]
```diff
@@ -44,8 +44,10 @@ jobs:
         uses: subosito/flutter-action@refs/pull/79/head
       - name: 'Install Tools'
         run: |
+          sudo apt update
+          sudo apt install -y clang-format
           flutter pub global activate melos
-          flutter pub global activate flutter_plugin_tools
+          pub global activate flutter_plugin_tools
```
Comment (stuartmorgan) [inline .github/workflows/all_plugins.yaml:50]: I highly recommend you pin a version here (see [here](https://github.com/flutter/packages/blob/master/.cirrus.yml#L9-L11) for instance). There isn't any kind of guarantee going forward that there won't be breaking changes to the tool's API.

Comment (mhadaily) [reply .github/workflows/all_plugins.yaml]: Good point, Done.

**Review by miquelbeltran — APPROVED** [review]
LGTM, there's a formatting issue detected that needs to be fixed before merging but otherwise good to go

**Pull request merged**